src/main/kotlin/uk/gov/justice/digital/hmpps/authorizationserver/data/repository/AuthorizationConsentRepository.kt

```diff
@@ -3,4 +3,6 @@ package uk.gov.justice.digital.hmpps.authorizationserver.data.repository
 import org.springframework.data.repository.CrudRepository
 import uk.gov.justice.digital.hmpps.authorizationserver.data.model.AuthorizationConsent
 
-interface AuthorizationConsentRepository : CrudRepository<AuthorizationConsent, AuthorizationConsent.AuthorizationConsentId>
+interface AuthorizationConsentRepository : CrudRepository<AuthorizationConsent, AuthorizationConsent.AuthorizationConsentId> {
+  fun findByPrincipalName(principalName: String): AuthorizationConsent?
```

Comment:
Would just use findById with the composite primary key `AuthorizationConsentId` instead of adding this